src/content/api/merchants.mdx

```diff
@@ -62,6 +62,10 @@ which define the payment methods available for a Payment Request.
     Physical Location of Merchant. It is highly recommended that you provide this otherwise, users won't be able to find you with our [Merchant Search](#search-merchants) API if they perform a origin + distance query.
   </Property>
 
+  <Property name="type" type="string">
+    Valid values: `physical` or `e-commerce`. Used for reporting purposes.
+  </Property>
+
```

Comment:
I think you need to explain that location can't be provided for an e-commerce merchant. I think that would probably go in the explanation for the location above. I think also explaining that e-commerce merchants will not show up on the merchant search would be important.

Reply:
I don't think there is a need to mention reporting, maybe you could just say something like type should be physical for any merchant that has a physical location. If a merchant is going to be used for e-commerce payments the type should be 'e-commerce'.

Reply:
This has been added to the model. See the code above.